utils/test/testDemoExamples.js

```diff
@@ -19,11 +19,13 @@ import React from 'react';
 import { mount } from 'enzyme';
 // $FlowFixMe
 import { ThemeProvider } from '@mineral-ui/component-utils';
+// $FlowFixMe
```

Comment:
You can get rid of these `$FlowFixMe`'s for `react-live`.  You just need to create the stub using `flow-typed`.

```sh
./node_modules/.bin/flow-typed create-stub react-live@latest
npm run flow:restart
```